src/components/course/rate.vue

```diff
@@ -117,27 +117,45 @@
       </v-row>
       <!-- 描述 -->
       <!-- 附件 -->
-      <v-row class="my-6 pl-3">
-
+      <v-row no-gutters class="mt-6">
+        <v-col col="12" lg="10" md="10" sm="12" xs="12">
+          <v-list-item-title class="text-h5 font-weight-medium">
+            Resources
+            <v-btn class="info ml-2" x-small @click="addfile">upload</v-btn>
+          </v-list-item-title>
+        </v-col>
+      </v-row>
 
-              <v-list-item-title class="text-h5 font-weight-medium">
-                Resources
-                <v-btn class="info ml-2" x-small @click="addfile">upload</v-btn>
-              </v-list-item-title>
-              <v-list-item-subtitle
-                v-for="fileUpload in fileUploads"
-                :key="fileUpload.id"
+      <v-row no-gutters class="mt-6">
+        <v-col col="12" lg="10" md="10" sm="12" xs="12">
+          <v-layout row wrap>
+            <v-card
+              outlined
+              v-for="fileUpload in fileUploads" :key="fileUpload.id"
+              max-width="212"
+              class="mt-1 ml-1"
               >
-                <a :href="fileUpload.imageUrl" target="_blank" v-if="userLoggedIn"
-                  >{{ fileUpload.type }} - {{ fileUpload.filename }}</a
-                >
-                <a @click="signupVisible = true;" v-else
-                  >{{ fileUpload.type }} - {{ fileUpload.filename }}
-                </a>
-                
-              </v-list-item-subtitle>
-
-
+              <v-list-item>
+                <v-list-item-avatar tile>
+                    <v-img :src='"../../assets/fileicon/"+fileUpload.fileType+".svg"' ></v-img>
+                </v-list-item-avatar>
+                <v-list-item-content>
+                  <v-list-item-title class=" mb-2 font-weight-medium">
+                    {{fileUpload.type}}
+                  </v-list-item-title>
+                  <v-list-item-subtitle>
+                    <a :href="fileUpload.imageUrl" target="_blank" v-if="userLoggedIn"
```

Comment:
maybe we should move this to v-list-item: a :href="fileUpload.imageUrl"
So that the user can hovers/clicks on the picture instead of having to click on the file name(which can be small sometimes)